frontend/src/metabase/admin/settings/components/widgets/SettingTextInput/SettingTextInput.tsx

```diff
@@ -21,6 +21,7 @@ export interface SettingInputProps {
   type: "text" | "password";
   normalize?: (value: Value) => Value;
   disabled?: boolean;
+  "data-testid"?: string;
```

Comment:
as commented somewhere else, I think we should move this test to a separate PR which contain the test that actually uses this prop.

Reply:
ah I was writing the tests with test IDs before realizing I should just use the `findByRole` stuff you wrote before. So I can just remove this